### DISCUSSION TITLE
footer_card color

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1274] [Q&A] [answered]
would it be useful to have a color param for the footer_card like the header_card?

When header's color is set to card, the footer's visibility seems odd when compared to the header and the body.

<img width="1785" alt="image" src="https://user-images.githubusercontent.com/64787868/156361358-c6b1a55d-4e05-4445-a001-73ba4be82bc0.png">

**Comment by mturoci** [comment, marked as answer]
If you want your footer to have card color, simply use `ui.form_card` instead of `ui.footer_card`.
